Update gridstack.js to 0.6.4.

ttill opened the pull request:
<!-- The following line must be included in your pull request -->
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

adumesny commented on package.json:
```diff
@@ -28,7 +28,7 @@
     "ember-cli-babel": "^7.11.1",
     "ember-cli-htmlbars": "^4.0.0",
-    "gridstack": "^0.6.2",
+    "gridstack": "^0.6.3",
```
Hey glad you guys and updating and keeping up with the latest rev! Yeah.
one thing to noticed is that your package does NOT need to list jquery and jquery-ui as dependencies - I will bring those in my package (there is work to remove that lib as well in the future).
Also **gridstack-all.min.js** will now include a **much smaller jqeury-ui** so you shouldn't include it anyway.

ttill replied:
Thanks for the hint!

adumesny commented:
glad to see keeping up with my gridstack changes.
Note that of 0.5.3 grid options `width` is now `column`, `height` now `maxRow`, and `setGridWidth()` now `setColumn()` to match what they are. Old names are still supported (console warnings).

https://github.com/gridstack/gridstack.js/releases/tag/v0.5.3

Also a much smaller jquery-ui is now include in gridstack-all.min.js so no need to include it anymore (hidden, to be removed later)

adumesny commented on index.js:
```diff
@@ -7,33 +7,8 @@ module.exports = {
-
     // Gridstack
-    [ 'gridstack', 'gridstack.jQueryUI' ].forEach(function(module) {
+    [ 'gridstack', 'gridstack.jQueryUI', 'jquery-ui' ].forEach(function(module) {
```
you may want to import in order (and one missing)
gridstack-poly.js (tiny file we need for IE11)', 'gridstack.js', 'jquery-ui.js', 'gridstack.jQueryUI.js

there is ALSO gridstack-extra.css (line #17 below) IF you need to support [2-11] columns (and min version of all files). Might want to update that as well.

ttill commented:
@adumesny Thank you very much for your hints and your ongoing work on this great library!

balajimanoharan commented:
Hey @ttill, thanks for working on this. there is still a build error that needs to be addressed before this can be merged. Do you mind taking a look at that?

adumesny commented:
https://travis-ci.org/yahoo/ember-gridstack/jobs/646774452?utm_medium=notification&utm_source=github_status
> not ok 13 Chrome 80.0 - [109 ms] - Integration | Component | grid stack: onChange action

note that I greatly reduced the number of calls to `change` callbacks, but apparently I introduced a bug where there is some case it doesn't get called when it should - see https://github.com/gridstack/gridstack.js/issues/1142
I'll try to fix that next week and release a 0.6.4 if I can (working on major 1.0.0 release with no jquery)

balajimanoharan commented:
Thanks @adumesny for letting us know. Looking forward to your 1.0 release as well.

ttill commented:
Unfortunately 0.6.4 did not fix the test. I did some experiments but was unable to figure out what's going on, as `onChange` is working as expected in my app.

adumesny commented:
@ttill doesn't look like the build is failing anymore though ? I don't see a red X next to change #
(but a conflicting package file)

balajimanoharan commented:
@adumesny The build didn't run in this case, since the package.json changes need to be resolved. But I built this locally, and noticed that adding new items to a grid does not trigger a `change` event anymore. Moving and resizing grid items do trigger the `change` event correctly.

adumesny commented:
yes adding/removing an item will ONLY now call change event on other nodes if hey changed, else you get just the add/remove with the latest position/size.

ttill commented:
I rebased on master and fixed the tests anticipating gridstack.js' new event logic and also considering `onAdded` and `onRemoved` in addition to `onChange`.

balajimanoharan commented:
Thanks @ttill @adumesny. Will publish a release later today.

adumesny commented:
great. I should have v1.0.0 later this week which removes jquery dependencies from API/package
That will require code and packaging change though.
